pkg/drivers/kic/kic.go

```diff
@@ -358,13 +364,34 @@ func (d *Driver) Stop() error {
 	}
 
 	if err := killAPIServerProc(d.exec); err != nil {
-		glog.Warningf("couldn't stop kube-apiserver proc: %v", err)
+		glog.Infof("kill kube-apiserver proc: %v", err)
+	}
+
+	if err := killETCDProc(d.exec); err != nil {
```

Comment:
this feels out of place to me The kic driver should not have to know about etcd.

Reply:
this would only be used by kic drivers, since our base image init script meses with the SIG STOP, other drivers won't need to use this. I will add a comment why we need this.